src/Form/HydroponicsSettingsForm.php

```diff
@@ -170,13 +173,88 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         ]
       ];
     }
+    elseif ($current_status['error'] == TRUE) {
+      $form['status']['stop'] = [
+        '#type' => 'button',
+        '#value' => $this->t('Reset Service'),
+        '#ajax' => [
+          'callback' => [$this, 'stopRunningServiceCallback'],
+          'effect' => 'fade',
+          'wrapper' => 'hydroponics-status',
+          'method' => 'replace',
+        ]
+      ];
+    }
 
     $form['active'] =  [
       '#title' => 'Check to enabled Hydroponics Queue Background processing service wakeup during Drupal Cron.',
       '#type' => 'checkbox',
       '#required' => FALSE,
       '#default_value' => $active,
     ];
+
+    //Add parameters depending on processing type selected
+    //The array KEY is the right part of drush command, i.e. hydroponics for archipelago:hydroponics
+    $processing_options['hydroponics'] = "A single process for all queues";
```

Comment:
This requires a $this->t() in front of each Option text

Reply:
done